dialogue-target/src/main/java/com/palantir/dialogue/Endpoint.java

```diff
@@ -25,7 +28,23 @@
  * as well as the types of the request and response body.
  */
 public interface Endpoint {
-    void renderPath(Map<String, String> params, UrlBuilder url);
+    /**
+     * {@link #renderPath(ListMultimap, UrlBuilder)} is preferred, however at least one {@code renderPath} must be
+     * implemented to avoid infinite recursion.
+     */
+    default void renderPath(Map<String, String> params, UrlBuilder url) {
```

Comment:
Just to reason through this: existing conjure clients will only have the ```renderPath(Map)``` implementation. So when BaseUrl#render comes around, it will call ```renderPath(ListMultimap)``` which will add a tiny bit of overhead with the MultimapAsMap wrapper, and call into ```renderPath(Map)``` implementation. I'm assuming you'll FLUP with a conjure change to implement the right method instead?

Reply:
Exactly!